### DISCUSSION TITLE
[Future idea] Do yearly blog post / evaluation about money situation

### DISCUSSION THREAD

**Discussion by inson1** [FossifyOrg/General-Discussion#14] [Ideas]
1. How much money did the project get last year?
2. What that enough?
3. How much more would you need?
4. How long can you continue from project from "war chest", if there is any?

I think not enough informations about this, was part of the problem with SMT.

**Comment by darkdragon-001** [comment]
https://opencollective.com/collectives could provide all of this transparency by default.

**Reply by inson1** [reply]
@darkdragon-001 Is possible to link paypal, crypto, patreon etc. to them?

**Reply by darkdragon-001** [reply]
They support credit cards, bank transfers and Paypal as far as I know. One can also manually add income and expenses.

**Reply by inson1** [reply]
@darkdragon-001 In that case it sounds great :)

**Comment by Ponchale** [comment]
We are a FOSS organization, we have forked all the applications but we would like to speak with the leaders so that we can reach an agreement and a union.

**Reply by tswistak** [reply]
If you care about FOSS community, why Midori Browser isn't updated on F-Droid? Google Play has the last update from October this year, when F-Droid version wasn't updated for 2 years. I really wouldn't like to see SMT forks go this way, that they would be accessible only via Play store...

Links for reference:
- https://f-droid.org/packages/org.midorinext.android/
- https://play.google.com/store/apps/details?id=org.midorinext.android

EDIT:

Is it somehow related to these two reports on F-Droid? I don't know so much their ecosystem, so I don't know when they stop building apps and how to find the reason. I can only see that it's flagged to have tracking anti-features.

- https://gitlab.com/fdroid/fdroiddata/-/issues/2412 (reporting to ad company)
- https://gitlab.com/fdroid/fdroiddata/-/issues/2317 (sending bug reports without consent)

**Reply by Ponchale** [reply]
Friend, let's not speak from ignorance, or misinformation, there are more reference links.

https://news.itsfoss.com/midori-11/
https://alternativeto.net/news/2023/11/midori-11-0-released-with-improved-performance-and-integration-with-other-astian-services/
https://ubunlog.com/midori-11-ya-fue-liberado-y-estas-son-sus-novedades/

What's happening, we were working on Midori's migration to Gecko and Geckoview we wanted to abandon everything from Google including Webview and after this we will return to F-Droid. Of course, if we ever say that we are not in favor of F-Droid or that Midori has died, we are not here to generate conflict but to do collaborative work and to move forward with all this.

![Screenshot_20231203_150334](https://github.com/FossifyX/General-Discussion/assets/285103/f4b346e5-73be-44ea-b978-c4bd101a61f1)

**Reply by tswistak** [reply]
I hope to see the latest Midori version on F-Droid. Before it happens, I wouldn't call a misinformation the obvious fact that I've written - Play store version is updated, F-Droid not.

**Reply by shuvashish76** [reply]
@Ponchale Wrong thread, IMO those who want to talk about organization & other related things should contact the owner "Naveen Singh" through mail, discuss then make an public announcement. Discussing publicly here with users leads nowhere.

**Reply by inson1** [reply]
@shuvashish76 Bad discussion, but it should be here. We should be part of that proccess imo.

**Reply by inson1** [reply]
@Ponchale Why not just contribute like normal contibutors? Or what could you do more? And why not first finish your big rewrite?

**Reply by Ponchale** [reply]
@inson1 Just listen to the proposal we have and let's align our points of view and find a way to work together, we do not seek to generate conflict, quite the opposite, we will make contributions, resources but let's talk, I have never managed to understand how a person or a group of people can judge without listening. to the other person. That's not it, I want to talk and present my point of view on how from Astian we can contribute, how we can support what resources we can give and what we want. I haven't said anything else, just talk to work together.

No one has asked, Ponchale, what is the proposal, you only want attacks and prejudices, not my friends, I don't think that is the way to receive someone who only wants to talk and propose good within the community.

**Reply by inson1** [reply]
@Ponchale I asked. "Or what could you do more?" I didnt attacked. I think the questions are good. You have big commitment and lots of work elsewhere, so question about that is good :) 

And the question "Why not just contribute like normal contibutors?" was about why do you think you are better then average contributor? not in bad meaning - who are you? how many are you? what do want? what do you have? Money? Skills and time? Fame? Idea? Something else? I dont know you. All I know abot you is that you have browser and are middle of big rewrite, which is big time hungry commitment

But sorry if that sounded harsh or like I was creating conflict.

**Reply by naveensingh** [reply]
@Ponchale Thank you for reaching out. I see you guys have planned to maintain a SimpleMobileTools fork as well. (Astium Apps suite?)

> we can reach an agreement and a union.

I don't mean any disrespect but I would prefer Fossify to stay as an independent open-source project without any ties to existing organizations (FOSS or not). I want Fossify to be known first as a community-supported feature-complete collection of reliable open-source apps and second as a fork of SimpleMobileTools and that's about it. We as a community will happily accept any contributions and support and we _will_ mention all our contributors and supporters on our project page, blogs, and releases.

**I urge you guys to not divide the community's time and resources into two parts but instead, if you can, consider contributing code or supporting Fossify through other means.**

Maybe I wouldn't have started Fossify if I knew another FOSS organization was already on it but I'd already started Fossify a day before [you notified us](https://github.com/SimpleMobileTools/General-Discussion/issues/241#issuecomment-1837605444).

Lastly, sorry for the late response, I would have found it sooner if you'd created a thread for discussing this directly.

**Reply by naveensingh** [reply]
If any community members seeing my comment above disagree with my decision, then please [create a poll](https://github.com/orgs/FossifyX/discussions/new?category=polls).

**Comment by naveensingh** [comment]
> I think not enough informations about this, was part of the problem with SMT.

@inson1 **I don't want to discuss it (please)** but after managing the apps for years, I think Tibor just didn't want to do it anymore. If it was a monetary issue, he probably would have quit sooner but I don't know. He's like a friend from afar. 

> money situation
> How much money did the project get last year?
> What that enough?
> How much more would you need?

I'll maybe do it. Obviously, I'm not here to be profitable but if all goes well, it would be good to slowly transition to a full-time maintainer position without worrying as if I'm doing something wrong (as the kids say, new fear unlocked).

> 4. How long can you continue from project from "war chest", if there is any?

I can kind of answer this right now, I'll be here full-time for the next 2-3 weeks but after that, I'll go back to reality. That is, after 2-3 weeks I'll be here mostly in my free time doing bug fixes and reviewing PRs. I hope to finish the project setup and release the apps before that.

**Reply by Ponchale** [reply]
@naveensingh Well, I appreciate your words, it is unfortunate that we cannot talk but it is okay, it is understandable taking into account everything that has happened, we will continue with the work and development of Astian Suite, taking into account that we try to follow the regular procedure.

And of course it goes without saying that everything we do has always been, always will be and will continue to be FOSS, if in the future they decide to collaborate, excellent here will always be the open door where we will first listen and then the decision will be made.
